index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+     <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+
+     </head>
+     <body>
+       <header>
+         <div class="head-wrapper">
+           <nav class="nav-bar">
+             <ul>
+               <li class="logo"><a href="#">My Logo</a></li>
+            </ul>
+            <div class="menu">
+                   <div class="toogle-btn"></div>
+                   <div class="toogle-btn"></div>
+                   <div class="toogle-btn"></div>
+             </div>
+           </nav>
+         </div>
+       </header>
+
+       <section>
+         <div class="headline-wrapper">
+          <div class="headline-container">
+            <div class="intro">I'm Vana Glad to see you!</div>
+            <div class="message">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+            <div class="end-message">
+              <h3>LET'S CONNECT</h3>
+            </div>
+          </div>
+
+          <div class="social-icon">
+            <div class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></div>
+            <div class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></div>
+            <div class="social-media"><a href="#"><i class="fa fa-medium"></i></a></div>
+            <div class="social-media"><a href="#"><i class="fa fa-github"></i></a></div>
+            <div class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></div>
+           </div>
+         </div> 
+       </section> 
```

Comment:
- [ ] Good job :clap: :clap:  I can see you are improving your design. However, I think it would be nice if you can follow the same `paddings` and `margins` colors and `font-sizes` that are used in the template. Check here point_down :point_down: 

your design | the template
--------------- | ----------------
![143390039-f6c2a839-f2f9-4217-acae-05cf29dcefb2](https://user-images.githubusercontent.com/65068771/143435936-2c7eb81c-13b2-43c2-b449-77ae83affe45.png) | ![Screenshot from 2021-11-25 13-46-48](https://user-images.githubusercontent.com/65068771/143436264-b2b32885-109e-4734-8f41-6df0335e8b8c.png)